### PR TITLE
Use Example Code Service for samples

### PR DESCRIPTION
**Pull request by 2m**

### CODE REVIEW

File: akka-docs/rst/dev/multi-node-testing.rst [hunk 1]
```diff
@@ -195,8 +195,9 @@ message send/receive.
 .. includecode:: ../../../akka-remote-tests/src/multi-jvm/scala/akka/remote/sample/MultiNodeSample.scala
   :include: package,spec
 
-The easiest way to run this example yourself is to download `Lightbend Activator <http://www.lightbend.com/platform/getstarted>`_
-and open the tutorial named `Akka Multi-Node Testing Sample with Scala <http://www.lightbend.com/activator/template/akka-sample-multi-node-scala>`_.
+The easiest way to run this example yourself is to download the ready to run
+`Akka Multi-Node Testing Sample with Scala <@exampleCodeService@/akka-samples-multi-node-scala>`_
```
Comment (ktoso) [inline akka-docs/rst/dev/multi-node-testing.rst]: cool links :-)

**Review by patriknw — APPROVED** [review]
LGTM

File: akka-docs/rst/intro/getting-started.rst [hunk 2]
```diff
@@ -14,14 +14,11 @@ for Java 6 in case your project can not upgrade to Java 8 just yet. It also incl
 Getting Started Guides and Template Projects
 --------------------------------------------
 
-The best way to start learning Akka is to download `Lightbend Activator <http://www.lightbend.com/platform/getstarted>`_
-and try out one of Akka Template Projects.
-
```
Comment (johanandren) [inline akka-docs/rst/intro/getting-started.rst]: Shouldn't the header go as well, now that there is no content?

**Review by johanandren — APPROVED** [review]
I approve

**Pull request merged**